website/client-old/js/config.js

```diff
@@ -35,7 +35,7 @@ angular.module('habitrpg')
           if (isUserLoaded && notification.type === 'CRON') {
             // If the user is already loaded, do not show the notification, syncing will show it
             // (the user will be synced automatically)
-            $rootScope.User.readNotification(notification.id);
+            // $rootScope.User.readNotifications([notification.id]);
```

Comment:
From manual testing, I have seen that this is duplicate. The Cron notification is read by the noitificationCtrl .